Packs/PrismaCloudITAdmin/Integrations/Prisma_Cloud_IT_Admin/Prisma_Cloud_IT_Admin.py

```diff
@@ -0,0 +1,523 @@
+
+
+def verify_and_load_scim_data(scim):
+    try:
+        scim = json.loads(scim)
+    except Exception:
+        pass
+    if type(scim) != dict:
+        raise Exception("SCIM data is not a valid JSON")
+    return scim
+
+
+def map_scim(scim):
+    try:
+        scim = json.loads(scim)
+    except Exception:
+        pass
+    if type(scim) != dict:
+        raise Exception('Provided client data is not JSON compatible')
+
+    mapping = {
+        "userName": "userName",
+        "email": "emails(val.primary && val.primary==true).[0].value",
+        "first_name": "name.givenName",
+        "last_name": "name.familyName",
+        "id": "id",
+        "timezone": "timezone",
+    }
+    parsed_scim = dict()
+    for k, v in mapping.items():
+        try:
+            parsed_scim[k] = demisto.dt(scim, v)
+        except Exception:
+            parsed_scim[k] = None
+    return parsed_scim
+
+
+''' COMMAND FUNCTIONS '''
+
+
+def test_module(client, args):
+    """
+    Returning 'ok' indicates that the integration works like it is supposed to. Connection to the service is successful.
+
+    Args:
+        client: PrismaCloudAdmin client
+        args  : PrismaCloudAdmin arguments passed
+
+    Returns:
+        'ok' if test passed, anything else will fail the test.
+    """
+    args
+    uri = 'user/id'
+    res = client.http_request(method='GET', url_suffix=uri)
+    if res.status_code == 200 or res.status_code == 400:
+        return 'ok', None, None
+    else:
+        raise Exception(f"{res.status_code} - {res.text}")
+
+
+def get_user_command(client, args):
+
+    scim = verify_and_load_scim_data(args.get('scim'))
+    scim_flat_data = map_scim(scim)
+    user_id = scim_flat_data.get('id')
+    username = scim_flat_data.get('userName')
+    email = scim_flat_data.get('email')
+
+    if not (user_id or username or email):
+        raise Exception('You must provide either the id,, email or username of the user')
+
+    if user_id:
+        user_term = user_id
+    else:
+        user_term = username if username else email
+
+    user_term = user_term.lower()
+    res = client.get_user_profile(user_term)
+
+    if res.status_code == 200:
+        res_json = res.json()
+        active = res_json['enabled']
+        generic_iam_context = OutputContext(success=True, iden=res_json.get('email'), email=res_json.get('email'),
+                                            username=username, details=res_json, active=active)
+    elif res.status_code == 400:
+        generic_iam_context = OutputContext(success=False, iden=user_id, username=username, errorCode=404,
+                                            errorMessage="User Not Found", details=res.headers.get('x-redlock-status'))
+    else:
+        generic_iam_context = OutputContext(success=False, iden=user_id, username=username,
+                                            errorCode=res.status_code,
+                                            errorMessage=res.headers.get('x-redlock-status'),
```

Comment:
does the `x-redlock-status` hold the error message in this case?

Reply:
Yes. It will provide the required error message